### PR TITLE
Fix Node SDK

### PR DESCRIPTION
**Pull request by adisbanda**

### CODE REVIEW

File: .github/workflows/build-and-test.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v2
         with:
-          node-version: '12.x'
+          node-version: '14.x'
```
Comment (adisbanda) [inline .github/workflows/build-and-test.yml:10]: Github Actions were failing because of this

File: package.json [hunk 2]
```diff
@@ -25,7 +25,7 @@
     "run-example": "cd example && yarn && yarn dev",
     "clean": "rm -rf dist",
     "lint": "eslint '**/*.{js,jsx,ts,tsx}'",
-    "prepare": "yarn pkg"
+    "prepare": "yarn pkg && yarn cjs"
```
Comment (adisbanda) [inline package.json:28]: This got the package to work locally in my Node app

File: tsconfig.json [hunk 3]
```diff
@@ -17,7 +17,8 @@
     "baseUrl": "./src",
     "paths": {
       "@/*": ["./*"]
-    }
+    },
+    "skipLibCheck": true
```
Comment (adisbanda) [inline tsconfig.json:21]: This skips type checking in the `node_modules` directory. The `exclude` a few lines below isn't working.

File: package.json (hunk 2 above)
Comment (adisbanda) [reply package.json:28]: The `cjs` directory was missing from our NPM package but visible in the segment one:

image

**Review by ferrucc-io — APPROVED** [review]
nice one!

**Pull request merged**